### ISSUE TITLE
The template root only one element. but vue-language-server show me the error.

### ISSUE THREAD

**Issue by zheung** [vuejs/vetur#508]
- [x] I have searched through existing issues
- [x] I have read through docs at https://vuejs.github.io/vetur/
- [x] I have read FAQ: https://github.com/vuejs/vetur/wiki/FAQ

## Info

- Platform: Windows 10
- Vetur version: 0.10.1
- VS Code version: 1.18.0-Insider

## Problem

`vue-language-server` show me the Error Message:
`The template root requires exactly one element.`

But I'm sure my code only has one root element. Also without webpack error.

It only happen when root element is "Frame" and with slot element.

my code:
````vue
<template>
	<Frame class="compCardBox">
		<Frame class="card" v-for="(card, ci) of cards" :key="ci">
			<Icon class="iconBox" :card="card" size="100" />
			<div class="nameBox">
				<div class="inCard">{{card.name}}</div>
				<div class="inCard">{{card.attr}}</div>
			</div>
		</Frame>
	</Frame>
</template>
````

bad, "Frame" and slot
![b1](https://user-images.githubusercontent.com/894004/32264403-7e28c52c-bf19-11e7-8919-01084fa19d6d.png)

good, "Frameeeee" and slot
![b2](https://user-images.githubusercontent.com/894004/32264410-82dbfe18-bf19-11e7-81ff-2e4fcdbd4034.png)

good, "Frame" without slot
![b3](https://user-images.githubusercontent.com/894004/32264412-8a5ec74c-bf19-11e7-9037-c999b31a1c03.png)

**Comment by zheung** [comment]
sorry that i never use `<frame>` tag in html, `<frame>` hsa no end tag.
in my code `Frame` is a compoment, they has same name.

**Comment by octref** [comment]
Yes, and since html tags are case-insensitive, I'm afraid you have to choose another name.

**Comment by HerringtonDarkholme** [comment]
I think this is a valid bug in eslint-plugin-vue.

**Comment by KoenPaas** [comment]
I encountered this issue and need to use the Frame component due to Nativescript's syntax, can you reopen this issue?

**Comment by techiedod** [comment]
Why is this closed when it's still an issue?
